### PR TITLE
Update notices.json

### PR DESCRIPTION
**Pull request by jgilliam17**
notices test

### CODE REVIEW

**Comment by jgilliam17** [comment]
@jaydeland not sure why it's failing, can you please take a look. Thanks

**Comment by jaydeland** [comment]
@jgilliam17 - If you click on the details above for the failed check you should be able to login with oneLogin and see the failure details.
 ajv validate -s notices.schema.json -d notices.json
notices.json invalid
[
  {
    keyword: 'additionalProperties',
    dataPath: '[1].conditions',
    schemaPath: '#/additionalProperties',
    params: { additionalProperty: 'ClientType' },
    message: 'should NOT have additional properties'
  }
]
ERROR: Job failed: command terminated with exit code 1

**Comment by jgilliam17** [comment]
Thanks for sharing the error
I see `This site can’t be reached` when checking for details

**Comment by jaydeland** [comment]
> Thanks for sharing the error
> I see `This site can’t be reached` when checking for details
@jgilliam17 
Oh sorry - forgot to mention you need to be on the VPN

File: notices.json [hunk 1]
```diff
@@ -1,29 +1,28 @@
...
     }
   },
   {
-    "id": "6",
+    "id": "8",
     "conditions": {
-      "audience": "all",
-      "clientType": "all"
+      "audience": "Member",
+      "ClientType": "web"
```
Comment (jgilliam17) [inline notices.json:21]:
```suggestion
      "clientType": "web"
```

**Comment by jaydeland** [comment]
@jgilliam17 - here is the error:
$ ajv validate -s notices.schema.json -d notices.json
notices.json invalid
[
  {
    keyword: 'enum',
    dataPath: '[1].conditions.audience',
    schemaPath: '#/definitions/Audience/enum',
    params: { allowedValues: [Array] },
    message: 'should be equal to one of the allowed values'
  }
]
ERROR: Job failed: command terminated with exit code 1

**Comment by jaydeland** [comment]
@jgilliam17 - note you can run the json linter locally if you like by install ajv-cli:
```bash
npm install -g ajv-cli
```
And then you can run:
```bash
ajv validate -s notices.schema.json -d notices.json
```

File: notices.json [hunk 2]
```diff
@@ -17,12 +17,12 @@
   {
     "id": "8",
     "conditions": {
-      "audience": "all",
-      "clientType": "all"
+      "audience": "Member",
```
Comment (jgilliam17) [inline notices.json:20]:
```suggestion
      "audience": "member",
```

**Pull request merged**